frontend/src/components/camp/picasso/Picasso.vue

```diff
@@ -29,12 +29,19 @@ Listing all given activity schedule entries in a calendar view.
       :weekdays="[1, 2, 3, 4, 5, 6, 0]"
       color="primary"
       :event-ripple="false"
-      @mousedown:event="entryMouseDown"
-      @click:event="entryClick"
```

Comment:
Wieso verwendest du nicht mehr den Click Event? Click Event ist viel zugänglicher als mousedown und mouseup.

Reply:
Wir hatten vorher 2 Logiken drin um clicks zu registrieren, einmal hier über den click event, und einmal wie mousedown mouseup. Ich habs nun konsolidiert in mousedown/moseup.

Die Logik im mousedown/mouseup kann darauf reagieren, wie viel ich den Mauszeiger zwischen down und up bewegt habe.

Im click event habe ich das nicht. Der Event wird immer dann getriggert, wenn ich mousedown und mouseup irgendwo auf dem schedule entry habe, selbst wenn ich den Mauszeiger dazwischen um 100px bewegt habe.

Unter Umständen aber hinfällig, je nach Resultat der Diskussion unten. Ich würd sagen, let's discuss am nächsten Meeting.